maps/torch/items/summarydocuments.dm

```diff
@@ -1,393 +1,375 @@
+/obj/item/documents/generic/corp/lpainfo	
+	name = "LPA reminder"	
+	desc = "An \"internal use only\" corporate document informing corporate liasions of the loss prevention associate's job aboard the ship. It cites concerns about the misuse of the LPA and general mistrust towards them from the rest of the crew, and warns how these tensions will damage EXO's image if they keep acting in this manner. It reminds them that LPAs are from an established industry and should have experience in many aspects other than carrying a firearm. It specifies good observation skills, solid restraint, an understanding of escalation rules in regards to their position, and that their experience as an industry professional can be useful as an advisor to their tasked liasion."
 
 /obj/item/documents/generic/corp/exointerest
 	name = "EXO and you"
-	desc = "\"Confidential\" corporate documents directed to the Corporate Liasion, informing them that their job is to represent EXO and their interests, to ensure the SEV Torch is able to maintain the currently defined direction that is within the interests of EXO as a whole. "
-//this one needs to be made torch specific
-///obj/item/documents/generic/corp/exointerest2
-//	name = "RE; Exo and you"
-//	desc = "\"Confidential\" memo directed to all Helios Corporate Liasions that highlights several instances of Corporate Liasons going above their charge on Helios project vessels. It reinforces that Corporate Liasons are not aboard the vessels to make decisions on behalf of the EXO board of directors, and ends by requesting Liasions re-read the original memo titled \"Exo and you\"."
+	desc = "Some \"confidential\" corporate documents directed to the corporate liasion, informing them that their job is to represent EXO and its immediate interests, which are to ensure the SEV Torch is able to maintain the currently defined direction that is within the interests of EXO as a whole. "
+
+/obj/item/documents/generic/corp/exointerest2
+	name = "RE: EXO and you"
+	desc = "A \"confidential\" memo directed to all Helios corporate liasions that highlights several instances of corporate liasons going above their authority on Helios project vessels. It reinforces that corporate liasons are not aboard the vessels to make decisions on behalf of the EXO board of directors, and ends by requesting that all liasions re-read the original memo titled \"EXO and you\"."
```

Comment:
Exointerest2 was commented out because it does not apply to all Helios CL's, as it says.
Only the Torch has this setup.

It would require a rewrite to be Torch-specific to be applicable.